### PR TITLE
add queue throughput metrics

### PR DESCRIPTION
**Pull request by briangough**
### Description

Add queue throughput metrics

Also adds a metric on setDoc bytes

#### Screenshots

NA

#### Related Issues / PRs

tbc

### Review

Adds metrics only

#### Potential Impact

Low, but any exception would crash docupdater.

#### Manual Testing Performed

- [X] Unit and acceptance tests 

#### Accessibility

NA

### Deployment

NA

#### Deployment Checklist

NA

#### Metrics and Monitoring

Make a queue dashboard for the new metrics.

#### Who Needs to Know?

NA

### CODE REVIEW

File: app/coffee/RealTimeRedisManager.coffee [hunk 1]
```diff
@@ -27,6 +28,8 @@ module.exports = RealTimeRedisManager =
 				catch e
 					return callback e
 				updates.push update
+				# record metric for updates removed from queue
+				metrics.summary "redis.pendingUpdates", jsonUpdate.length, {status: "pop"}
```
Comment (das7pad) [inline app/coffee/RealTimeRedisManager.coffee]: Let's track the metric before the try block -- it might bail out in the catch section.

Comment (das7pad) [reply app/coffee/RealTimeRedisManager.coffee]: Better yet, use a second loop before this one.

Comment (briangough) [reply app/coffee/RealTimeRedisManager.coffee:32]: It should be unlikely but I'll make a separate loop for simplicity (originally had it like that, then decided to combine them!).

**Comment by briangough** [comment]
Have updated the loop and added a metric for set/del on the doclines.

**Comment by briangough** [comment]
Will fix conflict in package.json later.

**Pull request merged**